experimental/saiexperimentaldashroutinggroup.h

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) 2014 Microsoft Open Technologies, Inc.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License"); you may
+ *    not use this file except in compliance with the License. You may obtain
+ *    a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    THIS CODE IS PROVIDED ON AN *AS IS* BASIS, WITHOUT WARRANTIES OR
+ *    CONDITIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT
+ *    LIMITATION ANY IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS
+ *    FOR A PARTICULAR PURPOSE, MERCHANTABILITY OR NON-INFRINGEMENT.
+ *
+ *    See the Apache Version 2.0 License for specific language governing
+ *    permissions and limitations under the License.
+ *
+ *    Microsoft would like to thank the following companies for their review and
+ *    assistance with these files: Intel Corporation, Mellanox Technologies Ltd,
+ *    Dell Products, L.P., Facebook, Inc., Marvell International Ltd.
+ *
+ * @file    saiexperimentaldashroutinggroup.h
+ *
+ * @brief   This module defines SAI extensions for DASH ROUTING GROUP
+ *
+ * @warning This module is a SAI experimental module
+ */
+
+#if !defined (__SAIEXPERIMENTALDASHROUTINGGROUP_H_)
+#define __SAIEXPERIMENTALDASHROUTINGGROUP_H_
+
+#include <saitypesextensions.h>
+
+/**
+ * @defgroup SAIEXPERIMENTALDASH_ROUTING_GROUP SAI - Experimental: DASH ROUTING GROUP specific API definitions
+ *
+ * @{
+ */
+
+/**
+ * @brief Attribute ID for dash_routing_group_routing_group
+ */
+typedef enum _sai_routing_group_attr_t
+{
+    /**
+     * @brief Start of attributes
+     */
+    SAI_ROUTING_GROUP_ATTR_START,
+
+    /**
+     * @brief Action set_routing_group_attr parameter ADMIN_STATE
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_ROUTING_GROUP_ATTR_ADMIN_STATE = SAI_ROUTING_GROUP_ATTR_START,
```

Comment:
I somehow remember this should be changed to disabled instead of admin state. But somehow it is missing?

Reply:
Changed to disabled